lacework-agent/values.schema.json

```diff
@@ -310,21 +310,6 @@
                         }
                     }
                 },
-                "codeaware": {
```

Comment:
probably this too

Reply:
Correct, and the values.yaml change

Reply:
<img src="https://media0.giphy.com/media/qgri3D9sTwCUGMcT8L/giphy.gif"/>